tensorbay/label/attributes.py

```diff
@@ -421,3 +422,36 @@ def dumps(self) -> Dict[str, Any]:
             contents["parentCategories"] = self.parent_categories
 
         return contents
+
+
+class AttributesInfo(NameOrderedDict[AttributeInfo]):
+    """This class defines the concept of AttributesInfo."""
+
+    _T = TypeVar("_T", bound="AttributesInfo")
+
+    def _loads(self, contents: List[Dict[str, Any]]) -> None:
+        self._data: "OrderedDict[str, AttributeInfo]" = OrderedDict()
```

Comment:
why init `_data` here?

Reply:
use `AttributesInfo.loads` will not not call `__init__`, so init `_data` here is needed.